config.txt

```diff
@@ -9,3 +9,5 @@ device_tree_address=0x1f0000
 device_tree_end=0x200000
 dtoverlay=miniuart-bt
 dtoverlay=upstream-pi4
+dtoverlay=vc4-kms-v3d-pi4
```

Comment:
I believe this is not needed because the DT nodes for a vc4 and v3d are already present in the `upstream-rpi4.dtbo` ?

We just need a firmware release that includes that config snippet since the latest one doesn't.

Reply:
Could we get a 1.33 release please @jlinton ?

Reply:
From https://github.com/raspberrypi/firmware/tree/master/boot/overlays:

> upstream-pi4:  Comprises the vc4-kms-v3d-pi4 and dwc2 overlays.

Therefore this Pull Request is not needed.

I'll see if I can produce a 1.33 release sometime this week...